### ISSUE TITLE
Duplicating HTML tags for FSX and IPYNB output.

### ISSUE THREAD

**Issue by DavideGCosta** [fsprojects/FSharp.Formatting#695]
I was trying to add HTML tags through markdown. The HTML output works fine, but the IPYNB and the FSX output duplicates the open tag. I may be missing something here but I think that this example should not duplicate the open tag for the .ipynb and .fsx output.

Input:
```fsharp
(**
 
<p> test
</p>
 
*)
```
```fsharp
.fsx and .ipynb output:
(**
<p> test
<p> test
</p>
*)
```

**Comment by DavideGCosta** [comment]
Also, you can see how the HMTL output looks:

```
<div class="container" id="fsdocs-content">
            <p> test
</p>
 
        </div>
```
It seems to be working fine, as markdown normally is supposed to allow embedded HTML to be passed through unchanged. 
But, in the .fsx and .ipynb output, as stated in the issue, there is a duplication of the open tag.

**Comment by dsyme** [comment]
Yes that looks like a bug

**Comment by dsyme** [comment]
One a quick glance over the code I think this duplicates the firstLine:

https://github.com/fsprojects/FSharp.Formatting/blob/master/src/FSharp.Formatting.Markdown/MarkdownParser.fs#L701

Might be correct if this

```fsharp
  | ((first, _n) as takenLine) :: TakeParagraphLines(html, rest) when first.StartsWith("<") ->
```

though it's written a bit inefficiently

**Comment by dsyme** [comment]
Will be fixed in 11.4.2 https://github.com/fsprojects/FSharp.Formatting/blob/master/RELEASE_NOTES.md
